third_party/tlx/tutorials/blackwell-fa-ws-pipelined-persistent_test.py

```diff
@@ -1284,7 +1286,6 @@ def _attn_bwd_ws(
         tl.float32,
         NUM_BUFFERS_TMEM,
         tlx.storage_kind.tmem,
-        reuse=dp_tiles,
```

Comment:
Removing this behavior improves performance because you no longer need to wait on the dq barrier. However, when I just set BLOCK_M1=64 this also introduced a correctness issue for H-DIM=128. I think the total memory should be the same for both buffers in this case, but this make sense to introduce a correctness issue? Do we need to debug this?

Reply:
I guess without reusing it'd OOM with BLOCK_M1=128? Does BLOCK_M1=64 always outperform BLOCK_M1=128?

Reply:
Can we guard this change under a condition, say BLOCK_M1=64, for now?

Reply:
Sure happy to do that.